mlflow/utils/validation.py

```diff
@@ -617,3 +620,14 @@ def _validate_trace_tag(key, value):
     key = _validate_length_limit("key", MAX_TRACE_TAG_KEY_LENGTH, key)
     value = _validate_length_limit("value", MAX_TRACE_TAG_VAL_LENGTH, value, truncate=True)
     return key, value
+
+
+def _validate_experiment_artifact_location_length(artifact_location: str):
+    max_length = MLFLOW_ARTIFACT_LOCATION_MAX_LENGTH.get()
+    if len(artifact_location) > max_length:
+        raise MlflowException(
+            "Invalid artifact location. The length of the artifact location cannot be "
```

Comment:
```suggestion
            "Invalid artifact location. The length of the artifact path cannot be "
```
Sounds a bit more precise